Compose-Ur-Pres/cup-gradle-plugin/src/main/kotlin/net/kodein/cup/gradle/CupDependencies.kt

```diff
@@ -9,4 +9,10 @@ public class CupDependencies internal constructor() {
         public val laser: String = "net.kodein.cup:cup-laser:${CupBuildInfo.VERSION}"
         public val speakerWindow: String = "net.kodein.cup:cup-speaker-window:${CupBuildInfo.VERSION}"
     }
+
+    public val widget: Widgets = Widgets()
+    public class Widgets internal constructor() {
+        public val material: String = "net.kodein.cup:cup-widget-material:${CupBuildInfo.VERSION}"
+        public val material3: String = "net.kodein.cup:cup-widget-material3:${CupBuildInfo.VERSION}"
```

Comment:
should be `cup-widgets-material3` (plural) as the artifact provides multiple widgets.